### DISCUSSION TITLE
btrack on GPU

### DISCUSSION THREAD

**Discussion by Ciro1990** [quantumjot/btrack#248] [Ideas]
Dear Alan,

first of all, thanks for creating and sharing this fantastic tool. I was wondering if btrack on GPU is now available and is working. 

Thanks a lot!
Ciro

**Comment by quantumjot** [comment]
Hi Ciro,
Haven't had much time to work on this unfortunately.  Is there a reason why you need the GPU accelerated version?

Related #39

**Comment by Ciro1990** [comment]
Hi Alan,

Thanks a lot for answering. The reason why I wanted to try the GPU accelerated version is because I have very large dataset. However, I realize the problem might also be in the setting up of the parameters for the tracking. Unfortunately, up until now I haven't managed to do so. Would it be possible for you to upload more examples of models for btracker?

Thanks again for the amazing work you and your lab do!
Ciro

**Comment by quantumjot** [comment]
Yes - arriving at a good configuration can be challenging. Are you able to share any extra details, like a log or a short movie (e.g. #125)? - I might be able to give you some quick advice.

**Comment by Ciro1990** [comment]
Yes of course. Find below the log of the Btracker and a video of the segmentation (cellpose) and the tracks created. For btracker I used the standard model (cell config) provided here on github:
![Log](https://user-images.githubusercontent.com/96627336/167639177-fe9a4e3a-df80-451d-bae8-835d07b47e65.JPG)

As you can see, tracks are created but due to the heterogeneity of movement of the cells, very often tracks for the same cell are split. 

https://user-images.githubusercontent.com/96627336/167641984-519fcfe7-57bd-402a-9117-a202c1d09f73.mp4

How could I improve this?
